src/Directory.Build.props

```diff
@@ -1,4 +1,10 @@
 ﻿<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <!-- Add IsImplicitlyDefined to any packages that were 'magically' referenced (RepoTools!). This
+         should be removed if RepoTools adds the metadata for their package references. -->
+    <PackageReference Update="@(PackageReference)" IsImplicitlyDefined="true" />
```

Comment:
What does `IsImplicitlyDefined` mean?

Reply:
https://docs.microsoft.com/en-us/nuget/tools/package-manager-ui
It means that a package is implicitly defined from an SDK. If set VS doesn't prompt to update it (since you can't) and Central Package Versioning won't error for having a version defined.

Reply:
Thanks.